templates/tram-hook/__name__/__name__.test.js

```diff
@@ -0,0 +1,7 @@
+import {{ name }} from './{{ name }}'
+
+describe('{{ name }}', () => {
+	it('should return true by default', () => {
+		expect({{ name }}()).toBe(true)
```

Comment:
What the hell is this syntax

Reply:
it's the template syntax, in reality it generates a file that looks like this:
![image](https://user-images.githubusercontent.com/326557/68532714-f0952680-02ee-11ea-82b1-e85014b3f2b6.png)

Reply:
Ahh, also nice font